tests/helpers/resize-container.js

```diff
@@ -15,7 +15,7 @@ export function timeout(milliseconds) {
 }
 
 
-export default async function resizeWindow(width, height) {
+export default async function resizeContainer(width, height) {
```

Comment:
I renamed the function and the file to be precise about what the test helper does. The test helper doesn't resize the (test) window, but resizes the container (i.e. `<ContainerQuery>` component).